src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: What is mixed content?
+authors:
+  - johyphenel
+  - rachelandrew
+date: 2019-09-07
+updated: 2020-09-16
+description: >
+  Mixed content occurs when initial HTML is loaded over a secure HTTPS connection,
+  but other resources are loaded over an insecure HTTP connection.
+tags:
+  - security
+---
+
+**Mixed content** occurs when initial HTML is loaded over a secure HTTPS connection,
+but other resources (such as images, videos, stylesheets, scripts)
+are loaded over an insecure HTTP connection.
+This is called mixed content
+because both HTTP and HTTPS content are being loaded to display the same page,
+and the initial request was secure over HTTPS.
+
+Requesting subresources using the insecure HTTP protocol weakens the security of the entire page,
+as these requests are vulnerable to **man-in-the-middle attacks**,
+where an attacker eavesdrops on a network connection and views or modifies the communication between two parties.
+Using these resources, an attacker can often take complete control over the page,
+not just the compromised resource.
```

Comment:
```suggestion
Requesting subresources using the insecure HTTP protocol weakens the security of the entire page,
as these requests are vulnerable to [**on-path attacks**](https://www.ietf.org/rfc/rfc7835.html#section-2.1.1),
where an attacker eavesdrops on a network connection and views or modifies the communication between two parties.
Using these resources, an attacker can often take complete control over the page,
not just the compromised resource.
```

FYI @rachelandrew we've got an internal "do not use" word list asking us to avoid `man-in-the-middle`. I don't expect you to memorize this (and we should automate this detection) but I figured I'd give you the FYI.